src/overlays/actors/ovl_En_Dog/z_en_dog.c

```diff
@@ -22,41 +28,576 @@ const ActorInit En_Dog_InitVars =
     (ActorFunc)EnDog_Update,
     (ActorFunc)EnDog_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Dog/func_809FAFD0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Dog/func_809FB038.s")
+static ColliderCylinderInit cylinderInit =
+{
+    0x06, 0x00, 0x09, 0x39,
+    0x10, 0x01, 0x00, 0x00,
+    0x00, 0x00, 0x00, 0x00,
+    0x00000000,
+    0x00, 0x00, 0x00, 0x00,
+    0xFFCFFFFF,
+    0x00, 0x00, 0x00, 0x00,
+    0x00, 0x01, 0x01, 0x00,
+
+    0x0010,
+    0x0014,
+    0x0000,
+    0x0000, 0x0000, 0x0000,
+};
+
+// There are multiple versions of sub98 init structs
+// Keeping these local until z_collision_check is decompiled
+typedef struct
+{
+    /* 0x00 */ u8 health;
+    /* 0x02 */ s16 unk_10;
+    /* 0x04 */ s16 unk_12;
+    /* 0x06 */ u16 unk_14;
+    /* 0x08 */ u8 mass;
+}sub98Init;
+
+static sub98Init sub98Data = 
+{
+    0x00,   //health
+    0x0000, //unk_10
+    0x0000, //unk_12
+    0x0000, //unk_14
+    0x32,   //mass
+};
+
+static struct_80034EC0_Entry animations[]=
```

Comment:
space before =?

Reply:
That will be taken care of by clang-format anyway.